examples/with-facebook-pixel/package.json

```
@@ -3,11 +3,15 @@
   "scripts": {
     "dev": "next dev",
     "build": "next build",
+    "lint": "next lint",
     "start": "next start"
   },
   "dependencies": {
     "next": "latest",
     "react": "^17.0.2",
     "react-dom": "^17.0.2"
+  },
+  "devDependencies": {
+    "eslint": "8.9.0"
```

```suggestion
    "start": "next start"
  },
  "dependencies": {
    "next": "latest",
    "react": "^17.0.2",
    "react-dom": "^17.0.2"
  },
```


I don't think this was a necessary change, could we revert it?

---

I can revert this if needed. When reading the template for contributing, I saw...
> Make sure the linting passes by running `yarn lint`
...which is why I added in the necessary packages for linting. Without those packages, `yarn lint` does not work. Let me know if you'd like me to revert adding the linting.

---

The Next.js repo has a linter already. :+1:

It's referring to that: https://github.com/vercel/next.js/blob/c7b2083f2eff850150ae6ac315d04ac74737e880/package.json#L26

---

Ah, got it! Sorry for the confusion. I'll revert those changes then.